src/public/translations/tw/translation.json

```diff
@@ -1237,7 +1237,7 @@
     "protected_session_timeout_description": "保護會話超時是一個時間段，超時後保護會話會從瀏覽器內存中清除。這是從最後一次與保護筆記的交互開始計時的。更多資訊請見",
     "wiki": "維基",
     "for_more_info": "更多資訊。",
-    "protected_session_timeout_label": "保護會話超時（秒）",
+    "protected_session_timeout_label": "Protected session timeout:",
```

Comment:
We shouldn't replace translations with their English counterpart as they would still count as translated. We need to remove them (or maybe replace them to an empty string) to be able to find them easily again.